server/channels/db/migrations/postgres/000136_create_attribute_view.up.sql

```diff
@@ -0,0 +1,11 @@
+DROP MATERIALIZED VIEW IF EXISTS AttributeView;
+
+CREATE MATERIALIZED VIEW AttributeView AS
```

Comment:
You can just do `CREATE MATERIALIZED VIEW IF NOT EXISTS`. There is no need to `DROP` before re-creating.

Reply:
Oh right, I tried and failed, I guess I put `if not exists` with wrong order. Thanks.